release/tune_tests/cloud_tests/app_config_ml.yaml

```diff
@@ -0,0 +1,17 @@
+base_image: "anyscale/ray-ml:nightly-py37-gpu"
```

Comment:
one thing amog and I discovered recently is that it's a lot of work to maintain all these app_configs.
so we have been pointing all the tests to a single canonical app_config.
don't know the situation here, just want to mention this option.